stixpy/map/stix.py

```diff
@@ -72,3 +75,45 @@ def wcs(self):
         # Validate the WCS here.
         w2.wcs.set()
         return w2
+
+    def to_hpc(self,
```

Comment:
This is a transform like any other so we should do it via the standard API can include this shortcut in the transformation code I think - need to think about this a bit more.

Reply:
OK, I think that makes sense.